template/en/default/global/user-error.html.tmpl

```diff
@@ -384,6 +384,10 @@
     [% title = "Comment Reactions Closed" %]
     This [% terms.bug %] no longer accepts comment reactions.
 
+  [% ELSIF error == "comment_reaction_restricted" %]
+    [% title = "Comment Reactions Restricted" %]
+    You are not allowed to react on this [% terms.bug %].
```

Comment:
Nit: Probably should be 'You are not allowed to react to comments on this [% terms.bug %].'

Reply:
Changed!